### PR TITLE
prebid integration guide

### PR DESCRIPTION
**Pull request by genwhittTTD**
prebid integration guide

### CODE REVIEW

File: docs/guides/integration-prebid.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+---
```
Comment (sunnywu) [inline docs/guides/integration-prebid.md:1]: which page should link to this? The publisher page's resource [section](https://unifiedid.com/docs/overviews/overview-publishers#resources)?

File: docs/guides/integration-prebid.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+Be sure to follow all the steps.
+
+## UID2 User ID Submodule
+
+UID2 requires initial tokens to be generated server-side. The UID2 module handles storing, providing, and optionally refreshing them. The module can operate in one of two different modes:
+
+- Client Refresh
+- Server-Only
+
+  *Server-Only* mode was originally called *legacy* mode, but it is a popular mode for new integrations where publishers prefer to handle token refresh server-side.
```
Comment (sunnywu) [inline docs/guides/integration-prebid.md:45]: let's remove _Legacy Mode_/Server-only mode here and below as Dana suggested for now to simplify? If you just want to merge this quickly then leave it unchanged it's fine.

File: docs/guides/integration-prebid.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+In planning your Prebid implementation, consider the following:
+
+- If you're trying to limit the size of cookies, provide the token in configuration and use the default option of local storage.
+
+- If you provide an expired identity, and the module has a valid identity which was refreshed from the identity you provide, the module uses the refreshed identity. The module stores the original token that it used for refreshing the token, and uses the refreshed tokens as long as the original token matches the token that you provided.
+
+- If you provide a new token that does not match the original token used to generate any refreshed tokens, all stored tokens are discarded and the new token used instead (refreshed if necessary).
+
+- During integration testing, you can set `params.uid2ApiBase` to `"https://operator-integ.uidapi.com"`. Be aware that you must use the same environment (production or integration) that you use for generating tokens.
```
Comment (sunnywu) [inline docs/guides/integration-prebid.md:187]: Is there a specific reason why you didn't copy 
__Parameter Descriptions for the usersync Configuration_ Section_ Section from the [readme]( https://github.com/prebid/Prebid.js/blob/master/modules/uid2IdSystem.md) to here given you have copied all other sections?

File: docs/guides/integration-prebid.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/guides/integration-prebid.md:1]: Yes. Done. Thx. Publisher overview and also integration guides summary.

File: docs/guides/integration-prebid.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/guides/integration-prebid.md:45]: @sunnywu I removed it. It's a chunk of info including code examples. The doc is very short now!

File: docs/guides/integration-prebid.md (hunk 3 above)
Comment (genwhittTTD) [reply docs/guides/integration-prebid.md:187]: @sunnywu  Really not sure. But, I've added it.

File: docs/guides/integration-prebid.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/guides/integration-prebid.md:1]: Publisher page and integration guides overview page. Thx.

File: docs/guides/integration-prebid.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/guides/integration-prebid.md]: Done. Removed a LOT of stuff that went with that.

File: docs/guides/integration-prebid.md (hunk 3 above)
Comment (genwhittTTD) [reply docs/guides/integration-prebid.md:126]: Not at all sure but I've added it now.

**Comment by genwhittTTD** [comment]
@sunnywu on your last comment. I think, at least for the moment, I'll leave it at prebid.org. Our docs will now come up on the search. Not ready to replace prebid.org as the main place to go, until we have the full set of requested improvements. Let me know if you disagree. This is still just a placeholder page essentially.

File: docs/guides/summary-guides.md [hunk 4]
```diff
@@ -15,6 +15,7 @@ The following guides provide integration instructions based on the needs and req
 | :--- | :--- |
 | [UID2 SDK for JavaScript Integration Guide](publisher-client-side.md) | This integration guide for publishers covers standard web integration scenarios that use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md). |
 | [Publisher Integration Guide, Server-Only](custom-publisher-integration.md) | This integration guide is for publishers that do not use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md). |
+| [Prebid Integration Guide](integration-prebid.md) | An integration guide for publishers who want to directly integrate with UID2 and generate identity tokens to be passed by Prebid in the RTB bid stream. |
```
Comment (sunnywu) [inline docs/guides/summary-guides.md:18]: given the way i heard from Kanishk - i think Prebid is the way most pubs will use to integrate with UID2. could we move this to the first row on the table if there's no other reason not to?

File: docs/overviews/overview-publishers.md [hunk 5]
```diff
@@ -43,6 +43,7 @@ The following documentation resources are available for publishers to implement
 | :--- | :--- | :--- | :--- |
 | Client-Side (Web) SDK | [UID2 SDK for JavaScript](../sdks/client-side-identity.md) | Client-Side JavaScript SDK that facilitates the process of establishing client identity using UID2 and retrieving advertising tokens for publishers. | Publishers |
 | Client-Side (Web) Integration Guide | [UID2 SDK for JavaScript Integration Guide](../guides/publisher-client-side.md) | This integration guide for publishers covers standard web integration scenarios that use the [UID2 SDK for JavaScript](/docs/sdks/client-side-identity.md). | Publishers |
+| Prebid | [Prebid Integration Guide](../guides/integration-prebid.md) | An integration guide for publishers who want to directly integrate with UID2 and generate identity tokens to be passed by Prebid in the RTB bid stream. | Publishers |
```
Comment (sunnywu) [inline docs/overviews/overview-publishers.md:46]: same here - move this line to top of the table?

File: docs/guides/summary-guides.md (hunk 4 above)
Comment (genwhittTTD) [reply docs/guides/summary-guides.md]: For sure. Didn't know it was the main integration method, gosh! Done.

File: docs/overviews/overview-publishers.md (hunk 5 above)
Comment (genwhittTTD) [reply docs/overviews/overview-publishers.md]: Done thx.

**Review by sunnywu — APPROVED** [review]

**Pull request merged**